lib/identity/client.js

```diff
@@ -1135,6 +1142,104 @@ class Client extends PartialClient {
     }
     return clientID
   }
+
+  /**
+   * createAccessRequest creates an access request
+   *
+   * @param {Object} accessRequest
+   *
+   * @return {Object} The created access request
+   */
+  async createAccessRequest(accessRequest) {
+    let response = await this.storage.authenticator.tsv1Fetch(
+      `${this.storage.config.apiUrl}/v1/identity/pam/access_requests/open`,
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: accessRequest.stringify(),
+      }
+    )
+    await checkStatus(response)
+    return AccessRequest.decode(await response.json())
+  }
+
+  /**
+   * searchAccessRequests allows for searching for all access requests associated or authorizable
+   * by the searcher, allowing for filtering based off group id to access or requestor id
+   *
+   * @param {Object} accessRequestSearchRequest
+   *
+   * @return {Object} Access requests matching the search filter
+   */
+  async searchAccessRequests(
+    filterByRequestorIDs = [],
+    filterByGroupIDs = [],
+    nextToken = 0,
+    limit = 1000
+  ) {
+    const requestParams = new AccessRequestSearchRequest(
+      filterByRequestorIDs,
+      filterByGroupIDs,
+      nextToken,
+      limit
+    )
+    let response = await this.storage.authenticator.tsv1Fetch(
+      `${this.storage.config.apiUrl}/v1/identity/pam/access_requests/search`,
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: requestParams.stringify(),
+      }
+    )
+    await checkStatus(response)
+    return AccessRequestSearchResponse.decode(await response.json())
+  }
+
+  /**
+   * describeAccessRequest fetches the current state of a single access request
+   *
+   * @param {Object} accessRequestID The id of the access request to describe
+   *
+   * @return {Object} An access request
+   */
+  async describeAccessRequest(accessRequestID) {
+    let response = await this.storage.authenticator.tsv1Fetch(
+      `${this.storage.config.apiUrl}/v1/identity/pam/access_requests/resource/${accessRequestID}`,
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      }
+    )
+    await checkStatus(response)
+    return AccessRequest.decode(await response.json())
+  }
+
+  /**
+   * deleteAccessRequest deletes a single access request
+   *
+   * @param {Object} accessRequestID The id of the access request to delete
+   *
+   * @return {Object} No content if deleted / non existant
+   */
+  async deleteAccessRequest(accessRequestID) {
+    let response = await this.storage.authenticator.tsv1Fetch(
+      `${this.storage.config.apiUrl}/v1/identity/pam/access_requests/resource/${accessRequestID}`,
+      {
+        method: 'DELETE',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      }
+    )
+    await checkStatus(response)
+    return {}
```

Comment:
i would just return `true` on success. it looks like there are other methods that return an object like
```js
{ success: true }
```
which may be more consistent.

the main point being that it's pretty convoluted to check for an empty object to know this succeeded: 
`Object.keys(response).length === 0`